code/modules/mob/living/silicon/robot/emote.dm

```diff
@@ -76,19 +73,19 @@
 			message = "<B>[src]</B> twitches violently."
 			m_type = 1
 
-		if ("twitch_s")
+		if ("twitch_s","twitches")

```

Comment:
if ("twitch", "twitches")
Oh, and @tigercat2000 will no doubt be along soon to mention the spacing between the options :wink: